content/en/tracing/profiler/connect_traces_and_profiles.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Investigate Code Hotspots from Traces
+kind: Documentation
+further_reading:
+    - link: 'tracing'
+      tag: 'Documentation'
+      text: 'APM Distributed Tracing'
+    - link: 'tracing/profiler/getting_started'
+      tag: 'Documentation'
+      text: 'Enable Continuous Profiler for Your Application.'
+    - link: 'tracing/profiler/intro_to_profiling'
+      tag: 'Documentation'
+      text: 'Introduction to Profiling'
+---
+
+If your application is showing performance problems in production, it's helpful to connect the distributed trace information from APM with the full profile of the code stack. Application processes that have both APM distributed tracing and continuous profiler enabled are automatically linked, so you can move directly from span information to profiling data on the Code Hotspots tab, and find specific lines of code related to performance issues.
+
+{{< img src="tracing/profiling/code_hotspots_tab.gif" alt="Code Hotspots tab shows profiling information for a APM trace span">}}
+
+## Prerequisites
+
+{{< programming-lang-wrapper langs="java,python" >}}
+{{< programming-lang lang="java" >}}
+Code Hotspots identification is enabled by default when you [turn on Profiling for your service][1]. For manually instrumented code, Continuous Profiler requires Scope activation of spans:
```

Comment:
is continuous profiler capitalized? you have it lowercase above

Reply:
I don't think scope is capitalized either

Reply:
Ah, yes, I checked with other profiler docs, and am going to go all lower-case, thank you.